<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConfigDevice</class>
 <widget class="QWidget" name="frmConfigDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnTest1">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Test1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTest2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Test2</string>
   </property>
  </widget>
  <widget class="QLabel" name="labTip">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>530</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
